manifests/0000_90_openshift-controller-manager-operator_04_cluster-policy-controller-role.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: cluster-policy-controller-leader-lock-cm
+  namespace: openshift-kube-controller-manager
+subjects:
+- kind: User
+  name: system:serviceaccount:openshift-infra:namespace-security-allocation-controller
+  apiGroup: rbac.authorization.k8s.io
+roleRef:
+  kind: Role
+  name: cluster-policy-controller-leader-lock
```

Comment:
do the role in this pull too.

Reply:
oh i see i named the file `role.yaml` instead of `rolebinding.yaml` fixing..

Reply:
done, renamed this to *rolebinding.yaml, moved role to this pull, added comments